pro/client/client.go

```diff
@@ -325,6 +352,8 @@ func (c *Client) execute(user common.UserConfig, method, path string, query url.
 		return err
 	}
 
+	req.Header.Set("Referer", "http://localhost:37457/")
```

Comment:
Since the redirect method is used by payment providers to redirect back to our app's success/fail states - the API expects you to pass 'Referer' header here